src/signals/IncidentMap/components/FilterPanel/FilterCategory.tsx

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: MPL-2.0 */
+/* Copyright (C) 2022 Gemeente Amsterdam */
+import { Checkbox } from '@amsterdam/asc-ui'
+
+import { StyledImg, LegendText, LegendItem } from './styled'
+
+export interface Props {
+  selected: boolean
+  text: string
+  onToggleCategory: (select: boolean) => void
+  icon?: string
+}
+
+export const FilterCategory = ({
+  selected,
+  text,
+  onToggleCategory,
+  icon,
+}: Props) => {
+  return (
+    <LegendItem htmlFor={text}>
```

Comment:
This looks like bad copy pasting. Cause this is not the Legend.

Reply:
changed the name